bazel/src/main/java/com/google/api/codegen/bazel/BazelBuildFileView.java

```diff
@@ -93,6 +93,15 @@
     }
     tokens.put("service_yaml", serviceYaml);
 
+    // We need to continue to supply the gapic_yaml to Java targets when the
+    // gapic_yaml is available, because that means it was added for some override.
+    String gapicYaml = "None";
+    String gp = bp.getGapicYamlPath();
```

Comment:
'gyp'? or maybe gapicYamlPath?

Reply:
`gapicYamlPath` it is. Thanks, my "Go" was showing through this the short identifier